components/permit-holders/PermitHolderHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Flex, HStack, Text } from '@chakra-ui/react'; // Chakra UI
+import { ChevronLeftIcon } from '@chakra-ui/icons'; // Chakra UI icon
+import Link from 'next/link'; // Link
+import RequestStatusBadge from '@components/internal/RequestStatusBadge'; // Request status badge
+import { Applicant } from '@lib/graphql/types'; // Applicant type
+
+type PermitHolderHeaderProps = {
+  readonly applicant: Applicant;
+  readonly applicationStatus: // TODO: Change this to the enum that we add
```

Comment:
This enum exists now (although I don't think it has all of these values... )

Reply:
Related comment: https://github.com/uwblueprint/richmond-centre-for-disability/pull/44#discussion_r684750450